packages/components/tag/src/tag.tsx

```diff
@@ -65,60 +67,68 @@ export type TTagProps = {
    * Content rendered within the tag
    */
   children: ReactNode;
+  /**
+   * Indicates the color scheme of the tag.
+   */
+  tone: 'primary' | 'warning' | 'surface';
 };
 
 const defaultProps: Pick<
   TTagProps,
-  'type' | 'isDisabled' | 'isDraggable' | 'horizontalConstraint'
+  'tone' | 'isDisabled' | 'isDraggable' | 'horizontalConstraint'
 > = {
-  type: 'normal',
+  tone: 'primary',
   isDisabled: false,
   isDraggable: false,
   horizontalConstraint: 'scale',
 };
 
 const Tag = (props: TTagProps) => {
-  const linkProps =
-    props.to && !props.isDisabled ? { as: Link, to: props.to } : {};
+  let tagBodyProps;
+
+  switch (true) {
+    case props.isDisabled:
+      tagBodyProps = {};
+      break;
+    case Boolean(props.to):
+      tagBodyProps = { as: Link, to: props.to };
+      break;
+    case Boolean(props.onClick):
+      tagBodyProps = { as: 'button' as ElementType };
+      break;
+    default:
+      tagBodyProps = {};
+  }
+
+  const isInteractive =
+    !props.isDisabled && (Boolean(props.onClick) || Boolean(props.to));
+
   return (
     <Constraints.Horizontal max={props.horizontalConstraint}>
       <div
         css={css`
           a {
-            cursor: pointer;
             text-decoration: none;
           }
           box-sizing: border-box;
-          cursor: default;
+          cursor: ${isInteractive ? 'pointer' : 'default'};
```

Comment:
should we also set `cursor: not-allowed` when `isDisabled` is true?

Reply:
There won't be anything in the markup that would indicate a possible action which was disabled (no `<a>` or `<button>` tag) but as a visual clue, why not?

Reply:
Let's just [try it out](https://github.com/commercetools/ui-kit/pull/2954/commits/8538bbb53e6d024801727046cab4d1d18455a599).